frontend/__tests__/reducers/features.spec.tsx

```diff
@@ -48,6 +48,7 @@ describe('featureReducer', () => {
       namespacedSet: null,
       safeResources: [],
       preferredVersions: [],
+      groupVersionMap: [],
```

Comment:
This name is confusing because it's an array, not a map.

I'd suggest using an ImmutableMap where the key is the API group name and the value is an array of versions